File: desktop/src/com/murilorcm/avidadoboitata/desktop/DesktopLauncher.java
package com.murilorcm.avidadoboitata.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.murilorcm.avidadoboitata.AVidaDoBoitata;

public class DesktopLauncher {
	public static void main (String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		new LwjglApplication(new AVidaDoBoitata(), config);
	}
}

File: desktop/src/br/com/murilorcm/avidadoboitata/desktop/DesktopLauncher.kt
package br.com.murilorcm.avidadoboitata.desktop

import br.com.murilorcm.avidadoboitata.Main
import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import java.awt.Toolkit

fun main(arg: Array<String>) {
    val config = LwjglApplicationConfiguration()
    val dimension = Toolkit.getDefaultToolkit().screenSize
    config.height = dimension.height
    config.width = dimension.width
    LwjglApplication(Main(), config)
}